CRM/Financial/BAO/Payment.php

```diff
@@ -525,13 +525,18 @@ private static function updateContributionStatus(int $contributionID, string $st
    *   and then assign apply that ratio to each line item.
    * - if overrides have been passed in we use those amounts instead.
    *
-   * @param $params
+   * @param array $params
+   * @param array $contribution
```

Comment:
Adding contribution to the signature here is not strictly required at this point but otherwise this PR conflicts with 3 open PRs that add it